package.json

```diff
@@ -41,7 +41,7 @@
     "@ampproject/filesize": "4.3.0",
     "@ampproject/google-closure-compiler": "20210406.0.0",
     "@ampproject/remapping": "1.0.1",
-    "@babel/core": "7.13.16",
+    "@babel/core": "7.14.0",
```

Comment:
Why are package files being touched by this PR? Bad merge?

In case it helps, here's what our dev docs suggest in order to keep your fork's `main` branch up to date: https://github.com/ampproject/amphtml/blob/main/contributing/getting-started-e2e.md#pull-the-latest-changes-from-the-amphtml-repository

Reply:
Yeah, bad merge